src/test/java/seedu/address/logic/commands/FindCommandIntegrationTest.java

```diff
@@ -0,0 +1,160 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalInternship.NINJA_VAN;
+import static seedu.address.testutil.TypicalInternship.PAYPAL;
+import static seedu.address.testutil.TypicalProject.ORBITAL;
+import static seedu.address.testutil.TypicalSkill.REACT;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.find.FindCommand;
+import seedu.address.logic.commands.find.FindInternshipCommand;
+import seedu.address.logic.commands.find.FindProjectCommand;
+import seedu.address.logic.commands.find.FindSkillCommand;
+import seedu.address.logic.commands.results.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.item.field.NameContainsKeywordsPredicate;
+import seedu.address.testutil.TypicalResumeBook;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code FindCommand}.
+ */
+public class FindCommandIntegrationTest {
+
+    private Model model;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(TypicalResumeBook.TYPICAL, new UserPrefs());
+    }
+
+    @Test
+    public void equals() {
```

Comment:
Equals test for integration maybe can look and compare between the different FindXXXCommand?